galley/pkg/runtime/state.go

```diff
@@ -0,0 +1,154 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package runtime
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/types"
+
+	"istio.io/api/config/mcp/v1alpha1"
```

Comment:
I forget if I already made this comment elsewhere, but can we try to use consistent package import aliases for protobufs, e.g. `mcp "istio.io/api/config/mcp/v1alpha1"`.

Reply:
I haven't updated the code with the review items from previous PRs. So please do keep mentioning them as you see them. :)